gui/wxpython/startup/utils.py

```diff
@@ -35,24 +35,29 @@ def get_possible_database_path():
     # potential translations (old Windows and some Linux)
     # but ~ and ~/Documents should cover most of the cases
     # ordered by preference and then likelihood
-    candidates = [
-        os.path.join(home, "grassdata"),
-        os.path.join(home, "Documents", "grassdata"),
-        os.path.join(home, "My Documents", "grassdata"),
-    ]
-    try:
-        # here goes everything which has potential unicode issues
-        candidates.append(os.path.join(home, _("Documents"), "grassdata"))
-        candidates.append(os.path.join(home, _("My Documents"), "grassdata"))
-    except UnicodeDecodeError:
-        # just ignore the errors if it doesn't work
-        pass
-    path = None
-    for candidate in candidates:
-        if os.path.exists(candidate):
-            path = candidate
-            break  # get the first match
-    return path
+    
+    options = ["grassdata","GRASSDATA","Grassdata","GrassData"]
```

Comment:
Please use PEP8 for formatting. In general, you can run Flake8 (`flake8 startup/utils.py`) to get some errors, but some are unfortunately still ignored/disabled including this one. Here, just remember _space after comma_ and use: `..."grassdata", "GRASSDATA"...`.